inclusive-design/idrc
# feat: replace project permalink field with slug field, restore missing order field, fix localization

## Description

- Adds a `slug` field to allow easier customization of project URLs
- Restores missing `order` field to project CMS editor to allow customizing project order
- Fix display of link to French versions of projects.

## Steps to test

1. Add a custom slug to a project; URL should use that instead of a slugified version of the title.
2. Edit the order of a project, it should appear at the top of the numeric order is lower.
3. Look at a project with a French translation; link to French version should say "Lire en Français".

## Additional information

Depends on https://github.com/fluid-project/eleventy-plugin-fluid/pull/218; using branch for now.

## Related issues

Resolves #721, and part of #719.
